services/github/github-release.service.js

```diff
@@ -8,19 +9,23 @@ import {
 } from './github-common-release.js'
 import { documentation } from './github-helpers.js'
 
+const extendedQueryParamSchema = Joi.object({
+  display_name: Joi.string().valid('tag', 'release').default('tag'),
```

Comment:
More then open to bikeshedding on names for this one, I just went for something quick to be able to implement the actual feature